docs/AUTHENTICATION.md

```diff
@@ -1,6 +1,6 @@
 # Intro
 
-This document is a guide on how to connect to CONNECT data services (Cds) using a client of your choice. We will first cover the difference between authorization (AuthZ) and authentication (AuthN) and how they relate to you as a developer of ADH. Then we will introduce some possible scenarios using some GitHub examples.
+This document is a guide on how to connect to CONNECT data services (CDS) using a client of your choice. We will first cover the difference between authorization (AuthZ) and authentication (AuthN) and how they relate to you as a developer of CDS. Then we will introduce some possible scenarios using some GitHub examples.
```

Comment:
Nitpick and I know this wasn't your change, but when first introducing an abbreviation like this `CONNECT data services` should be `CONNECT Data Services`

Reply:
I am wrong, after looking elsewhere (for example in the AVEVA docs) the correct name is `CONNECT data services`